CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+## [1.7.0] - 2024-09-23
+
+This release is the first after the donation of gocql to the Apache Software Foundation (ASF)
+
+### Changed
+- DRIVER_NAME parameter in STARTUP messages changed to new project URL after donation.  This should provide
```

Comment:
**suggestion**: Should the pull request number and/or Cassandra issue number be mentioned as well? For previous versions the pull request numbers were mentioned in case a user is interested in more details.

**nitpick**: Technically, the value in the STARTUP message is not a proper URL, as it lacks `//` in the beginning (if parsed as URL, everything goes to the path component). It is not a new module path either as we haven't changed the module path yet. Perhaps `new project path` or `new project name` would be more accurate?

Reply:
Good points both @martin-sucha!  I will absolutely add the PR numbers to the relevant entries and come up with better terminology.  Maybe something more generic like "identifier" makes more sense here?